sdk/js/src/svm.ts

```diff
@@ -250,6 +251,7 @@ function extractSwapInfo(swapOpportunity: OpportunitySvmSwap): {
   tokenProgramSearcher: PublicKey;
   tokenInitializationConfigs: TokenAccountInitializationConfigs;
   memo?: string;
+  userMintUserBalance: anchor.BN;
```

Comment:
why doesn't the server just provide the amount to wrap?

Reply:
we can't do this since it can depend on the bid amount (when the trade is searcher_specified)